### ISSUE TITLE
the train data about the model

### ISSUE THREAD

**Issue by zyg11** [zhou13/neurvps#22]
Hi zhou,  Thanks for sharing your code. I tested the model in the laneline dataset , the performance was better than a lot algorithms. I wander whether the number of train data is just 2000. Because the performance is amazing.

**Comment by zhou13** [comment]
Yes, and you should be able to reproduce it using the released code. (I think you are referring to the tmm17 model).
